WcaOnRails/spec/mailers/previews/job_failure_mailer_preview.rb

```diff
@@ -0,0 +1,13 @@
+# Preview all emails at http://localhost:3000/rails/mailers/job_failure_mailer
+class JobFailureMailerPreview < ActionMailer::Preview
+  def notify_admin_of_job_failure
+    job = Delayed::Job.new(id: 4242)
+    exception = nil
+    begin
+      raise "This is an error!"
+    rescue Exception => e
+      exception = e
+    end
+    JobFailureMailer.notify_admin_of_job_failure(job, exception)
```

Comment:
Here too
`let(:mail) { JobFailureMailer.notify_admin_of_job_failure(job, RuntimeError.new("This is an error!")) }`